Source/WebGPU/WebGPU/QuerySet.mm

```diff
@@ -136,3 +136,13 @@ void wgpuQuerySetSetLabel(WGPUQuerySet querySet, const char* label)
 {
     WebGPU::fromAPI(querySet).setLabel(WebGPU::fromAPI(label));
 }
+
+uint32_t wgpuQuerySetGetCount(WGPUQuerySet querySet)
+{
+    return WebGPU::fromAPI(querySet).queryCount();
+}
+
+WGPUQueryType wgpuQuerySetGetType(WGPUQuerySet querySet)
+{
+    return WebGPU::fromAPI(querySet).visibilityBuffer() ? WGPUQueryType_Occlusion : WGPUQueryType_Timestamp;
```

Comment:
I think we should make a QuerySet::getType() instead of doing logic here. I don't think the `wgpu` functions themselves should do anything at all except API marshaling.

Reply:
Good idea, I agree